server/odc-service/src/main/java/com/oceanbase/odc/config/ScheduleConfiguration.java

```diff
@@ -317,6 +317,22 @@ public ThreadPoolTaskExecutor queryProfileMonitorExecutor() {
         return executor;
     }
 
+    @Bean(name = "copilotQueryExecutor")
```

Comment:
1. why only thread pool, no implementation?

2. if copilot is included in odc, we may add a switch configuration, e.g. odc.features.copilot.enabled

Reply:
implementation in odc-cloud, switch configuration will be added after the ODC supports Copilot in the future

Reply:
if implemented in odc-cloud, why create thread pool here ?

better add CloudScheduleConfiguration in odc-cloud instead.

Reply:
ok